generator.go

```diff
@@ -47,7 +50,7 @@ const (
 )
 
 // GenerateRegistration asks the user questions and generates a config and registration based on the answers.
-func GenerateRegistration(asName, botName string, reserveRooms, reserveUsers bool) {
+func GenerateRegistration(asName, botName string, reserveRooms, reserveUsers bool) error {
```

Comment:
Not sure if this should return an error, since it's supposed to be interactive (it doesn't even return the registration).

If we decide to make it interactive, those few errors should be just shown to the user instead of returning them.

If we keep the return, the other returns in the function need to be updated to `return nil`